python/lsst/validate/drp/validate.py

```diff
@@ -37,6 +37,19 @@
 from .util import getCcdKeyName, repoNameToPrefix, calcOrNone
 from .io import saveKpmToJson
 
+from lsst.obs.cfht import MegacamMapper
```

Comment:
Sorry, @r-owen I didn't catch this in my earlier review!  I'm still developing my reviewing skills.

`validate_drp` does not depend on `obs_cfht`.  We can't assume that this mapper is available.

`objIdToCcdId` should be passed the mapper in some way.

This routine isn't currently used by anything here.  Did you mean to include it as part of DM-5005?

[I totally understand how this routine is useful to you.]